go.mod

```diff
@@ -28,7 +25,8 @@ require (
 
 require (
 	cosmossdk.io/errors v1.0.0-beta.7
-	cosmossdk.io/math v1.0.0-rc.0
+	cosmossdk.io/math v1.1.2
```

Comment:
and the cosmossdk math version

Reply:
similar

Reply:
[18a6add](https://github.com/celestiaorg/celestia-app/pull/2545/commits/18a6adda7940d679355188caba2c736f22b51248)